packages/auto-drive/src/api/calls/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './download.js'
+export * from './read.js'
```

Comment:
I still don't get why the path are in .js in a .ts file...
I get that ounce we build the package it build .js files, but these files should also be able to be use in native ts file, and so having .js path will brake this...

Reply:
Have been updated to `.ts` extension